changelog/@unreleased/pr-190.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: add AcceptFuncs lambda visitor
```

Comment:
Changelogs are meant to be consumed by end users as part of the release notes, so this description should be more extensive -- as it stands, I wouldn't be able to tell what this means if I were to just read the release notes without further context. The heading/first sentence can be a summary, but that should be followed with more details that explains what was added here in a manner that is understandable to someone reading the release notes.

As an example:

```
Generate an AcceptFuncs lambda visitor for union types.

Adds an option to generate an "AcceptFuncs" function for union types which accepts lambda functions that can be used as visitors. This allows visitors to be called on union types without having to implement a visitor interface.
```

Doesn't have to be exactly the above, but hopefully that provides an idea.

Reply:
Thanks, good catch